haproxy/haproxycfg.py

```diff
@@ -86,7 +87,7 @@ def _init_new_links():
             if ADDITIONAL_SERVICES:
                 additional_services = ADDITIONAL_SERVICES.split(",")
                 NewLinkHelper.get_additional_links(docker, additional_services, haproxy_container,
-                                                   links, Haproxy.cls_linked_services)
+                                                   links, Haproxy.cls_linked_services, cls_additional_services)
```

Comment:
Here will raise an error:INFO:haproxy:Error loading ADDITIONAL_SERVICES: global name 'cls_additional_services' is not defined